src/blocks/home/home.css

```diff
@@ -0,0 +1,26 @@
+.home {
+  background-image: url('../../images/home_pic.jpg');
```

Comment:
you can use like this `background: url('../../images/home_pic.jpg') center no repeat` without rules `background-position: center;
  background-repeat: no-repeat;`

Reply:
Done